[IMP] Recruitment: updating images and instructions for 17 (#7869)
odoo/documentation: content/applications/hr/recruitment.rst

   Otherwise, if a new stage is created, or an existing stage is modified, those changes are
   visible on all job positions.

.. _recruitment/settings:

Settings
========

Before creating a job position to be recruited for, the various settings for the *Recruitment* app
should first be configured. To view and edit the settings, navigate to :menuselection:`Recruitment
app --> Settings`. After any changes are made, click the :guilabel:`Save` button in the top-left
corner to save all the changes.

Job posting
-----------

The *Job Posting* section of the *Recruitment* app settings has only one selection to make. If job
positions are to be posted to the company's website, enable the :guilabel:`Online Posting` option.

This option only works if the *Website* application is also installed.

Recruitment process
-------------------

This section of the settings page specifies what the database can and cannot do during the
recruitment process.

Send interview survey
~~~~~~~~~~~~~~~~~~~~~

Odoo is capable of having a survey, or exam, sent to an applicant to better determine their skill
set. This aids the recruitment team and helps them determine which candidates should be moved

> through the recruitment pipeline, and which should no longer be considered.

Author: I love this idea! I linked ot the surveys essentials for now, and then when that doc is created it can be referenced instead.

Enable the :guilabel:`Send Interview Survey` option to be able to send surveys to applicants.

.. note::
   The *Surveys* application must be installed to use this feature.

Send SMS
~~~~~~~~

It is possible to send text messages to applicants directly through the *Payroll* app. To do so,

> enable the :guilabel:`Send SMS` option. This option requrtes credits to use. For more information,

Reviewer: Suggestion to similarly acknowledge the internal link that shows up when the feature is enabled, and to break off the last sentence from the paragraph.

```suggestion
enable the :guilabel:`Send SMS` option. This option requires credits to use, which can be obtained
by clicking the :guilabel:`➡️ Buy credits` internal link that emerges when the feature is enabled.

For more information,
```

refer to the :doc:`SMS pricing and FAQs
</applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.

.. _recruitment/cv-display:

CV display
~~~~~~~~~~

> When applicants submit an application, one of the default required fields is a resumé, or CV. All

Reviewer: think this is redundant?
```suggestion
When applicants submit an application, one of the required fields is a resumé, or CV. All
```

Reviewer: Also might be neat to add an **:abbr:** tag here to define **CV (curriculum vitae)** 😄

resumés are stored in the *Documents* application, and are accessible on the applicant's card.

A resumé has the option to appear on the applicant's card, instead of having a document link appear

> in the chatter, where it then needs to be expanded or downloaded to view it.

Reviewer: Specificity question around location: 
Is the CV available on the _card_ or to the right of the _form_, once the card is clicked into?

Author: It's the form (I thought if it as the selected card). It's fully explained now.

Enable the :guilabel:`CV Display` option to show the resumé on the applicant's card by default, and
in addition to the document link. When enabled, the resumé appears on the right side of the
applicant's card.

> If the window or tab is not wide enough, the resumé does not appear. The screen must be wide enough

Reviewer: Consider formatting this as a **note**, perhaps since it's supplemental information having to do with browser width.

to accommodate the resumé. The full-screen view typically accommodates this view.

.. image:: recruitment/cv-display.png
   :align: center
   :alt: The resumé on an applicant's card, appearing on the right side.

.. _recruitment/cv-ocr:

CV digitization (OCR)
~~~~~~~~~~~~~~~~~~~~~

> When an application is submitted, it is possible to have Odoo extract the applicant's name, phone

Reviewer: Extract *where*, exactly?
Into the applicant card? Contact record?

number, and email address directly from the resumé. To do so, enable the :guilabel:`CV Digitization
(OCR)` option.

When enabled, additional options appear. Click on the corresponding radio button to select one of
the following options:

- :guilabel:`Do not digitize`: this option turns off resumé digitization.

> - :guilabel:`Digitize on demand only`: this option only digitizes resumes when requested. A

Reviewer: Curious: is it possible to digitize in bulk, maybe in a list view group selection?

Author: I don't believe so- I tried myself by clicking and draggin, or importing, or creating new and adding several- looks lik eyu can only do one at a time. And online form the application form, only one. The ONYL thing I think could happen is if you emailed the email alias several applications at once, but I don't think that works- I am checking with the experts if there is a way to do that. Although between you, me, and the wall, since it imports such litte informatio (the bare mnimum) it almostdoesn't seem worth it, if the oly way is to email. But I'll update when I hear back! In the meantime, I expanded this a little.

Settings
========

Before creating a job position in Odoo, configure the necessary settings for the *Recruitment* app.
To view and edit the settings, navigate to :menuselection:`Recruitment app --> Settings`. After any
changes are made, click the :guilabel:`Save` button in the top-left corner to save all the changes.

Job posting
-----------

The *Job Posting* section of the *Recruitment* app settings has only one selection to make. If job
positions are to be posted to the company's website, enable the :guilabel:`Online Posting` option.

.. note::
   This option only works if the *Website* application is also installed.

Recruitment process
-------------------

This section of the settings page specifies what the database can and cannot do during the
recruitment process.

Send interview survey
~~~~~~~~~~~~~~~~~~~~~

Odoo is capable of having a survey sent to an applicant to gather more information about them.
Surveys can be thought of as exams, or questionnaires, and can be customized in various ways to
provide the recruitment team with valuable insights into the applicant

Enable the :guilabel:`Send Interview Survey` option to send surveys to applicants. Once enabled, an

> :guilabel:`➡️ Interview Survey` internal link button appears. Click the :guilabel:`➡️ Interview

Author: True- this was just suc an old PR, lol. I'll be sure to do all of those in the new format now!

   The applicant flow with all its stages are universal and applies to all job positions, unless
   specified. :ref:`A specific stage can be configured <recruitment/customize-stages>` to be
   job-specific, meaning that specific stage is only visible for that specific job position.
   Otherwise, if a new stage is created, or an existing stage is modified, those changes are
   visible on all job positions.

.. _recruitment/settings:

Settings
========

Before creating a job position in Odoo, configure the necessary settings for the *Recruitment* app.
To view and edit the settings, navigate to :menuselection:`Recruitment app --> Settings`. After any
changes are made, click the :guilabel:`Save` button in the top-left corner to save all the changes.

Job posting
-----------

The :guilabel:`Job Posting` section of the *Recruitment* app settings has only one selection to
make. If job positions are to be posted to the company's website, enable the :guilabel:`Online
Posting` option.

.. note::

>    The :guilabel:`Online Posting` only works if the *Website* application is also installed.

Reviewer: Good catch Sam.
Alternatively a descriptive noun would complete the sentence, after the gui

```suggestion
   The :guilabel:`Online Posting` feature only works if the *Website* application is also installed.
```

~~~~~~~~~~~~~~~~~~~~~

When an application is submitted using any of the available methods, such as an online application
submission, emailing a resume to the job position alias, or creating an applicant record directly
from the database, it is possible to have Odoo extract the applicant's name, phone number, and email
address from the resumé and populate the applicant's form. To do so, enable the :guilabel:`CV
Digitization (OCR)` option.

When enabled, additional options appear. Click on the corresponding radio button to select one of
the following options:

- :guilabel:`Do not digitize`: this option turns off resumé digitization.
- :guilabel:`Digitize on demand only`: this option only digitizes resumes when requested. A
  :guilabel:`Digitize document` buttons appears on applicant cards. When clicked, the resumé is
  scanned and the applicant's card is updated.
- :guilabel:`Digitize automatically`: this option automatically digitizes all resumés when they are
  submitted.

Beneath these options are two additional links. Click the :guilabel:`→ Buy credits` button to
purchase credits for CV digitization. Click the :guilabel:`→ View My Services` to view a list of all

> current services, and their remaining credit balances.

Author: I missed those little guys!

Select an :guilabel:`Email Template` from the list to be used when this refuse reason is selected.

If a new :guilabel:`Email Template` is desired, type in the name for the new template in the field.
Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
window appears.

Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.
Enter the email content in the :guilabel:`Content` tab. Make any other desired modifications to the
template, then click :guilabel:`Save & Close` to save the template and return to the
:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
Template` field.

To make any modifications to a refuse reason, click on the line, and make any desired changes.

New applicant
~~~~~~~~~~~~~

An applicant card can be manually added if needed. If an applicant needs to be added to the list of
prospective candidates and an applicant card has not been created yet (they have not applied for the
job online) an applicant card can be easily added from the job position Kanban view in one of two

> ways, using either the :ref:`Quick add <recruitment/quick-add-applicant>` icon or the :ref:`New
> <recruitment/create-new-applicant>` button.

Reviewer 2: There is no text on the UI stating "Quick add", so you can leave it lowercase and state it is a button, whereas the icon is the + sign
```suggestion
ways, using either the :ref:`quick add <recruitment/quick-add-applicant>` button or the :ref:`New
<recruitment/create-new-applicant>` button.
```

.. _recruitment/quick-add-applicant:

Quick add
*********

> Quickly add a new applicant using the :guilabel:`Quick add` icon. If not already in the job position

Reviewer 2: ```suggestion
Quickly add a new applicant using the *quick add* button. If not already in the job position Kanban
```

> Kanban view, navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app
> --> Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
> position card that the applicant should be added to. Then, click on the small :guilabel:`+ (plus
> sign) Quick add` icon in the top-right of the :guilabel:`New` stage to quickly add a new applicant.

Reviewer 2: ```suggestion
view, navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
position card that the applicant should be added to. Then, click on the small :icon:`fa-plus`
:guilabel:`(plus)` icon in the top-right of the :guilabel:`New` stage to quickly add a new
applicant.
```

  :guilabel:`Applicant's Name`, then the :guilabel:`Subject/Application Name` is what is displayed
  on the applicant card in the Kanban view.
- :guilabel:`Applicant's Name`: enter the applicant's name.
- :guilabel:`Email`: enter the applicant's email address.
- :guilabel:`Applied Job`: the current job position populates this field. If needed, the job
  position can be changed by selecting a different position from the drop-down menu. If a different
  job position is selected, after the card is created, the card appears in the selected job
  position.

After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
new blank applicant card appears. Click either the :icon:`fa-trash-o` :guilabel:`delete` icon or
anywhere on the screen to close the card.

If preferred, after entering the applicant name, click :guilabel:`Edit` and a detailed
:guilabel:`Applicant Form` loads. :ref:`Enter the information on the form
<recruitment/applicant-details>`.

.. image:: recruitment/quick-add.png
   :align: center
   :alt: All the fields for a new applicant form entered when using the Quick Add option.

.. _recruitment/create-new-applicant:

New
***

> Add a new applicant including all the relevant information using the :guilabel:`New` button. If not
> already in the Kanban view for the job position to add an applicant to, navigate to the main
> recruitment dashboard by going to :menuselection:`Recruitment app --> Applications --> By Job
> Positions`. Then, click the :guilabel:`New` button in the top-left of the Kanban view and a blank
> applicant form loads.

Reviewer 2: ```suggestion
Add a new applicant including all the relevant information using the *New* button. If not already in
the Kanban view for the job position to add an applicant to, navigate to the main recruitment
dashboard by going to :menuselection:`Recruitment app --> Applications --> By Job Positions`. Then,
click the :guilabel:`New` button in the top-left of the Kanban view and a blank applicant form
loads.
```

  applicant. The selections available in the drop-down menu correspond to the :guilabel:`Job
  Positions` configured on the main *Recruitment* dashboard.
- :guilabel:`Job Title`: the specific name of the position being offered to the applicant.
- :guilabel:`Department`: the :guilabel:`Department` the job position falls under.
- :guilabel:`Contract Start Date`: the date the contract takes effect. The default date is the
  current date. To modify the date, click the drop-down to reveal a calendar. Use the :guilabel:`<
  (left)` and :guilabel:`> (right)` arrows on either side of the month to navigate to the desired
  month, then click on the :guilabel:`day` to select the date.
- :guilabel:`Yearly Cost`: the annual salary being offered.
- :guilabel:`Link Expiration Date`: job offers are only valid for a specific period of time. The
  default expiration date is 30 days. Modify the expiration date, if desired.

Send offer
~~~~~~~~~~

When the information is all updated, click :guilabel:`Send By Email` to send the offer to the
applicant.

If the applicant does not have an email address listed on their applicant card, a warning appears in

> a red box at the bottom of the :guilabel:`Generate a Simulation Link` pop-up window, stating `The
> applicant does not have a valid email set. The Offer Link won't be able to be completed.` Click
> :guilabel:`Discard`, enter an email on the applicant's card, then click the :guilabel:`Generate
> Offer` button again.

Reviewer 2: Use guilabel for text displayed on the UI
```suggestion
a red box at the bottom of the :guilabel:`Generate a Simulation Link` pop-up window, stating
:guilabel:`The applicant does not have a valid email set. The Offer Link won't be able to be
completed.` Click :guilabel:`Discard`, enter an email on the applicant's card, then click the
:guilabel:`Generate Offer` button again.
```
